text/0000-buildpack-author-subteam.md

```diff
@@ -0,0 +1,125 @@
+
+As such, `libcnb` currently doesn't have well-defined processes around how it should be maintained and improved upon in order to increase adoption or serve the needs of existing users.  This leads to it feeling like an afterthought even though it is the only "official" language binding provided by the project. This lack of process hasn't stopped a [reasonable number](https://pkg.go.dev/github.com/buildpacks/libcnb?tab=importedby) of buildpack implementations from using it.
+
+Also, our user research has indicated that authoring a buildpack is difficult, and many potential buildpack authors are not go developers. In order to grow the number of buildpack authors there needs to be alternative language bindings whether provided by the project or fostered from the community:
+
+- [python](https://github.com/samj1912/python-libcnb)
+- [bash](https://github.com/jkutner/libcnb.bash)
+- [rust](https://github.com/Malax/libcnb)
+- [community go alternative](https://github.com/paketo-buildpacks/packit)
+
+Ultimately, there needs to be a group of people focused on addressing this need of making buildpack authorship more approachable.
+
+# What it is
+[what-it-is]: #what-it-is
+
+The high level charter of the proposed Buildpack Authors subteams is to help buildpack authors create buildpacks.
+
+More concretely the sub-team  would serve the following goals:
+1. Create a community focused on the needs of buildpack authors.
+1. Advocate for Buildpack Author needs from the community throughout the project
+1. Devote attention to improving our existing buildpack-author tooling (`libcnb`).
+1. Provide a home for any future buildpack author tooling (testing buildpacks and language buildpack templates).
+1. Encourage buildpack authors to get more involved in and contribute to the project by creating a team exclusively focused on building more shared tooling for buildpack authors.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+## Sub-team Guidelines
+
+The Buildpack Author sub-team will follow the same guidelines as other sub-teams:
+
+- include at least one core team member
+- responsible for narrower sets of concerns related to specific aspects of the project
+
+## Language Binding Responsibilties
+These are the responsibilities for maintaining any language binding:
+1. Implementing new APIs in a timely manner
+1. Release process, planning and updates
+1. Triaging issues and supporting users
+1. Documentation
+1. Buildpack Examples using these bindings
+1. Planning and executing improvements
+
+Today, this means taking ownership over the `libcnb` repo doing the above for it.
+
+## Inline Buildpack Responsibilites
+The buildpack sub-team is responsible for sheparding and maintaining [Inline Buildpacks](https://github.com/buildpacks/rfcs/blob/main/text/0048-inline-buildpack.md).
```

Comment:
What, concretely, does this mean at this point? Is this about the development of it? What about the pack and lifecycle aspects of inline buildpacks?

Reply:
I think in this case we meant to concentrate more around gathering feedback around the inline buildpacks API and trying to make it more user friendly over time. (Or possibly supplementing it with a dsl like packfile).

This again would need to be done in collaboration with the implementation and platform teams.

Reply:
If the implementation of inline buildpacks in is pack (the current plan) I don't think the word maintaining is appropriate here. Components have maintainers but features shouldn't.

Maybe this responsibility belongs in "Collaboration with other teams"?

Reply:
Sounds good to me. I do think it's more sheparding/championing issues/concerns which include some implementation peaces.

Reply:
I clarified this in d979f93. Let me know this makes sense.